content/release-10-16-0/devices-10-16-0-bundle/10-16-0-383.md

```diff
@@ -0,0 +1,36 @@
+---
+weight: 25
+title: Release 10.16.0.383
+layout: bundle
+---
+
+<!--10.16.0.377-10.16.0.383-->
+
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device management</td>
+<td>Availability calculations in the "Availability" overview and for individual devices were not correct. Percentages were calculated for the needed time plus an additional 24 hours. The time interval has now been corrected.</td>
```

Comment:
```suggestion
<td>Availability calculations in the "Availability" overview and for individual devices were not correct. Percentages were calculated for the needed time plus an additional 24 hours. The time interval has now been corrected.</td>
```

what's the correct interval?

maybe also replace "needed" with "required"

Reply:
Changed to be more clear